Formula/file-roller.rb

```diff
@@ -51,6 +69,6 @@ def post_install
   end
 
   test do
-    system "#{bin}/file-roller", "--version"
+    assert_match version.to_s, shell_output("#{bin}/file-roller --version 2>/dev/null")
```

Comment:
On balance I think it's probably sensible to not shove it to `/dev/null` in case it spits out a message one day that's actually useful and not just _"Hey, these upstream devs are using laughably outdated methods"_.